gatsby/src/components/Speaker.js

```diff
@@ -36,6 +36,9 @@ const Person = styled.div`
   }
   .avatar {
     height: 96px;
+    width: 96px;
+    object-fit: cover;
+    background-color: #eee;
```

Comment:
Avoid the content reflow when avatar image loads. Also show gray background if no image is present.

Reply:
Tested it and it's working.